src/app/Main/Body/QueryPage/Results/TableResults/TableContent-Table.component.tsx

```diff
@@ -52,21 +52,23 @@ interface ResultTableProps {
     isRowAndCol: boolean;
 }
 
-export function ResultsTableComponent(props: ResultTableProps): JSX.Element {
+export function TableContentTableComponent(
+    props: ResultTableProps

```

Comment:
ContentTableProps?

Reply:
Yes, it was changed in the commit "Unify names of interfaces"